Orchestrator/Features/Manifests/GetManifestForAsset.cs

```diff
@@ -0,0 +1,163 @@
+﻿using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using DLCS.Core;
+using DLCS.Core.Collections;
+using DLCS.Model.Assets;
+using DLCS.Web.Requests.AssetDelivery;
+using DLCS.Web.Response;
+using IIIF;
+using IIIF.ImageApi.Service;
+using IIIF.Presentation.V2;
+using IIIF.Presentation.V2.Annotation;
+using IIIF.Presentation.V2.Strings;
+using IIIF.Serialisation;
+using MediatR;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Orchestrator.Infrastructure.Mediatr;
+using Orchestrator.Models;
+using Orchestrator.Settings;
+
+namespace Orchestrator.Features.Manifests

```

Comment:
Looking forward a bit - one of the St Louis tasks is IIIF v3 for named queries (and single asset manifest). That's when the Handle method below will need to make a canvas appropriate to the asset type (it only works for images atm).

In future, if the asset is not an image and you ask for the v2 version, the DLCS should just redirect to the v3 version I think. It certainly won't do the IxIF fake stuff like Wellcome.

Reply:
Yeah - I'm not sure whether the request to this handler should have a `Version` param, or whether it should be 2 different request/handlers, one for each version, and the Controller can decide which to dispatch.